subsys/nrf_rpc/nrf_rpc_uart.c

```diff
@@ -98,6 +110,52 @@ static void ack_tx(struct nrf_rpc_uart *uart_tr, uint16_t ack_pld)
 	k_mutex_unlock(&uart_tr->ack_tx_lock);
 }
 
+static uint16_t tx_flip(struct nrf_rpc_uart *uart_tr, uint16_t crc_val)
+{
+	if (!IS_ENABLED(CONFIG_NRF_RPC_UART_RELIABLE)) {
+		return crc_val;
+	}
+
+	if (uart_tr->flips.tx_flip == FLIP_ZERO) {
+		crc_val &= 0x7fffu;
+		uart_tr->flips.tx_flip = FLIP_ONE;
+	} else {
+		crc_val |= 0x8000u;
+		uart_tr->flips.tx_flip = FLIP_ZERO;
+	}
+
+	return crc_val;
+}
+
+static bool rx_flip(struct nrf_rpc_uart *uart_tr, uint16_t crc_val)
```

Comment:
```suggestion
static bool rx_flip_valid(struct nrf_rpc_uart *uart_tr, uint16_t crc_val)
```
Can we try to assign some more descriptive name?